docs/plot.rst

```diff
@@ -0,0 +1,369 @@
+        "hydro",
+        "geothermal",
+        "other",
+        "dfo",
+        "ng",
+        "solar",
+        "wind",
+        "storage",
+        "solar_curtailment",
+        "wind_curtailment",
+      ]
+
+      plot_generation_time_series_stack(
+        scenario, "Western", resources, time_freq="D", normalize=True, t2c=t2c, t2l=t2l,
+        t2hc=t2hc
+      )
+
+  .. image:: img/single/generation_stack_western_ts.png
+
+
+Curtailment
+###########
+- plot renewable generators curtailment time series in an area (`notebook
+  <https://github.com/Breakthrough-Energy/PostREISE/blob/develop/postreise/plot/demo/plot_curtailment_time_series_demo.ipynb>`_)
+
+  .. code-block:: python
+
+      import matplotlib.pyplot as plt
+
+      from postreise.plot.plot_curtailment_ts import plot_curtailment_time_series
+      from powersimdata import Scenario
+
+      scenario = Scenario(403)
+
+      t2c = {"wind_curtailment": "blue", "solar_curtailment": "blue"}
+
+      plot_curtailment_time_series(
+        scenario, "Eastern", ["wind","solar"], time_freq='D', t2c=t2c,
+        label_fontsize=30, title_fontsize=35, tick_fontsize=25, legend_fontsize=25,
+      )
+      plt.show()
+
+  .. image:: img/single/curtailment_solar_eastern_ts.png
+  .. image:: img/single/curtailment_wind_eastern_ts.png
+
+
+Price
+#####
+- map locational marginal price
+
+  .. code-block:: python
+
+      from bokeh.io import show
+
+      from postreise.plot.plot_lmp_map import map_lmp
+      from powersimdata import Scenario
+
+      scenario = Scenario(3287)
+
+      map = map_lmp(scenario)
+      show(map)
+
+  .. raw:: html
+      :file: img/single/lmp_usa_map.html
+
+
+General
+#######
+- plot any time-series values using a heatmap where each column is one color-coded day
+
+  .. code-block:: python
+
+      import matplotlib.pyplot as plt
+      from powersimdata import Scenario
+
+      from postreise.analyze.generation.curtailment import (
+        calculate_curtailment_time_series,
+      )
+      from postreise.plot.plot_heatmap import plot_heatmap
+
+      scenario = Scenario(3287)
+      curtailment = calculate_curtailment_time_series(scenario).sum(axis=1)
+
+      plot_heatmap(
+          summed_curtailment,
+          cmap="PiYG_r",
+          scale=1e-3,
+          cbar_label="GW",
+          vmin=0,
+          vmax=250,
+          cbar_tick_values=[0, 50, 100, 150, 200, 250],
+          cbar_tick_labels=['0', '50', '100', '150', '200', '≥250'],
+          time_zone="ETC/GMT+6",
+          time_zone_label="(CST)",
+          contour_levels=[250],
+      )
+
+  .. image:: img/single/curtailment_usa_heatmap.png
+
+- map transmission lines color coded by interconnection
+
+  .. code-block:: python
+
+      from bokeh.io import show
+
+      from postreise.plot.plot_interconnection_map import map_interconnections
+      from powersimdata import Scenario
+
+      scenario = Scenario(3287)
+      grid = scenario.get_grid()
+
+      map = map_interconnections(grid)
```

Comment:
I'd rename this since `map` is a builtin. Ideas: `line_map`, `transmission_map`, `ic_map`, `interconnect_map`, not sure what sounds best.

Reply:
Done